src/main/java/io/prometheus/wls/rest/ExporterServlet.java

```diff
@@ -69,7 +69,9 @@ private void displayMetrics(WebClient webClient, MetricsStream metricsStream, MB
                 sort(metrics).forEach(metricsStream::printMetric);
         } catch (RestQueryException e) {
             metricsStream.println(
-                  withCommentMarkers("REST service was unable to handle this query\n" + selector.getPrintableRequest()));
+                  withCommentMarkers("REST service was unable to handle this query\n"
+                      + "exception: " + e.getMessage() + '\n'
```

Comment:
Please reverse the order of this so that "this query" is followed by query itself.